pkg/ibmvpc/sg_analysis_test.go

```diff
@@ -107,3 +110,101 @@ func TestSGRule(t *testing.T) {
 	require.Equal(t, sgRule.index, 1)
 	require.Equal(t, ruleStr, "index: 1, direction: inbound,  conns: protocol: all, remote: 0.0.0.0/0, local: 10.240.10.0/32\n")
 }
+
+func TestNoLocalField(t *testing.T) {
+	cidr1t, _ := ipblock.FromCidrOrAddress("10.250.10.1")
+	cidr2t, _ := ipblock.FromCidrOrAddress("10.250.10.0/30")
+
+	c1 := connection.TCPorUDPConnection(netp.ProtocolString("UDP"), 5, 87, 10, 3245)
+	c2 := connection.TCPorUDPConnection(netp.ProtocolString("TCP"), 1, 100, 5, 1000)
+
+	rulesTest1 := []*SGRule{
+		{
+			target:      cidr1t,
```

Comment:
```suggestion
			target:      newIPBlockFromCIDROrAddressWithoutValidation("10.250.10.1"),
```
I think the test will be more readable this way, when IPBlock objects generated directly on test fields.
Check the other fields as well.